ui/store/initial-state.js

```diff
@@ -0,0 +1 @@
+export default { openedFootnote: null };
```

Comment:
Do we want to add a comment explaining why some elements of the state, such as the parts of it reflected in the URL, aren't currently represented here?  Just in case folks come here expecting to see a bunch more stuff?

Reply:
Can do.